cmd/create/cluster/cmd.go

```diff
@@ -80,7 +80,8 @@ const (
 	MinReplicasSingleAZ = 2
 	MinReplicaMultiAZ   = 3
 
-	listInputMessage = "Format should be a comma-separated list."
+	listInputMessage          = "Format should be a comma-separated list."
+	listBillingAccountMessage = " To see the list of billing account options, you can use interactive mode by passing '-i'"
```

Comment:
```suggestion
	listBillingAccountMessage = "To see the list of billing account options, you can use interactive mode by passing '-i'"
```

minor nit. It's kind of strange to have the space in the beginning, probably better to have the space in the places where it needs to use it.